config/jobs/kubernetes-sigs/cloud-provider-azure/cloud-provider-azure-config.yaml

```diff
@@ -492,7 +492,7 @@ presubmits:
     extra_refs:
       - org: kubernetes-sigs
         repo: cluster-api-provider-azure
-        base_ref: release-1.12
+        base_ref: release-1.13
```

Comment:
Please skip upgrading jobs with templates in cloud-provider-azure. They don't have the cloud-provider-azure CAAPH change. Let me update those templates and upgrade CAPZ version later.
E.g. pull-cloud-provider-azure-e2e-ccm-vmss-ip-lb-capz
If the job uses templates in CAPZ, such job is fine.
E.g. pull-cloud-provider-azure-e2e-ccm-vmss-capz

/hold

Reply:
thx @lzhecheng, reverted all jobs w/ a `CLUSTER_TEMPLATE` config to `release-1.12`

PTAL

Reply:
Thank you Jack. But it seems for some jobs, you forget to keep CAPZ template version align with CAPZ version.

Reply:
done